### DISCUSSION TITLE
[Refine] Change the Data Provider result data from a Rest API with response format

### DISCUSSION THREAD

**Discussion by Jaylord-Bucayu** [refinedev/refine#5998] [Q&A] [answered]
**I have a rest api that formatted like this :**
{
    "status": true,
    "path": "/v1/products",
    "statusCode": 200,
    "data": [
        {
            "creditPrice": 0,
            "_id": "66555690ca7da4a27034621d",
            "name": "Updated",
            "price": 25.2,
            "isActive": true,
            "description": "",
            "images": [],
            "categories": [
                {
                    "isActive": true,
                    "_id": "66553df2f20d34aa27365942",
                    "title": "Sample",
                    "description": "sample description",
                    "code": "CAT-2P0KIXVXE",
                    "createdAt": "2024-05-28T02:14:10.910Z",
                    "updatedAt": "2024-05-28T02:14:10.910Z"
                }
            ],
            "createdAt": "2024-05-28T03:59:12.077Z",
            "updatedAt": "2024-05-28T03:59:12.077Z"
        },
        
    ]
}

**How could I   change the data provider to suite in per data fetching? Thanks in Advance!**

**Comment by BatuhanW** [comment, marked as answer]
Hey @Jaylord-Bucayu you can swizzle simple-rest data provider and modify it according to your needs.

https://refine.dev/docs/packages/cli/#commands
